eo-runtime/src/main/eo/org/eolang/collections/list.eo

```diff
@@ -60,8 +80,17 @@
         acc
 
   # Reduce from start "a" using the function "f"
+  [a f] > reduced
+    ^.reducedi > @
+      a
+      [a idx item]
+        &.f > @
+          a
+          item
+
+  # DEPRICATED, please remove after update of eo-hamcrest.
```

Comment:
@levBagryansky I think it would be better to use [pdd](https://www.yegor256.com/2010/03/04/pdd.html) to create a new issue, instead of comment. Just add `@todo` tag and provide short description about new task

Reply:
Ok, I have just edited